appengine/standard_python3/bundled-services/blobstore/django/main_test.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import re
+import subprocess
+
+import pytest
+import requests
+
+
+@pytest.fixture
+def version():
+    """Launch a new version of the app for testing, and yield the
+    project and version number so tests can invoke it, then delete it.
+    """
+
+    output = subprocess.run(
+        "gcloud app deploy --no-promote --quiet --format=json",
+        capture_output=True,
+        shell=True,
+    )
+
+    try:
+        result = json.loads(output.stdout)
+        version_id = result["versions"][0]["id"]
+        project_id = result["versions"][0]["project"]
+    except Exception as e:
+        print(f"New version deployment output not usable: {e}")
+        print(f"Command stderr is '{output.stderr}'")
+        raise ValueError
```

Comment:
Can we pass `output.stderr` to this?

(It doesn't look like that gets captured in our CI systems, e.g. [here](https://source.cloud.google.com/results/invocations/9fa7329f-3926-4ced-81fd-2689e411ce62/targets/github%2Fpython-docs-samples%2Fappengine%2Fstandard_python3%2Fbundled-services%2Fblobstore%2Fdjango/tests).)